components/form/form-element-mixin.js

```diff
@@ -227,6 +227,18 @@ export const FormElementMixin = superclass => class extends LocalizeCoreElement(
 		await this.updatedComplete;
 	}
 
+	reset() {
+		this.invalid = false;
+		this.validationError = null;
+		this._errors = [];
```

Comment:
Let me know if there's anything else we should be resetting here.

Reply:
Does this reset the element _value_ as well, or just its error state? When I think about what "resetting a form" would do, I think I'd expect it to act like `<button type="reset">` where it also resets values to their "initial" state.

I'm not sure whether the case we're solving for requires a full reset, but if not maybe we should call this "resetValidation"?

Reply:
It wouldn't reset the values. Agreed we're just solving for a validation reset so I'll rename.